### PR TITLE
Features/docs facade attributes

### PR DESCRIPTION
**Pull request by jnnr**
Addresses #128.

### CODE REVIEW

**Review by c-moeller — APPROVED** [review]
Great, thank you. I added the text from the facades docstring also to the general documentation, because there was the same text.

File: docs/getting_started.rst [hunk 1]
```diff
@@ -17,12 +17,13 @@ optimization).
...
-the hood the facade then uses these arguments to construct an oemof.solph-component and sets it up
-to be easily used in an :py:obj:`EnergySystem`. In oemof.thermal each technology has a facade class,
+Facades are classes that offer a simpler interface to more complex classes. More specifically, the :class:`Facade` s
+in this module inherit from `oemof.solph`'s generic classes to serve as more concrete and energy specific interface.
+The concept of the facades has been derived from oemof.tabular. The idea is to be able to
+instantiate a :class:`Facade` using only keyword arguments. Under the hood the :class:`Facade` then
+uses these arguments to construct an `oemof.solph` component and sets it up to be easily used in an
+:py:obj:`EnergySystem`. Usually, a subset of the attributes of the parent class remains while another
+part can be addressed by more specific or simpler attributes. In oemof.thermal each technology has a facade class,
 that can be found in the module oemof.thermal.facades. See the
```
Comment (jnnr) [inline docs/getting_started.rst]:
```suggestion
part can be addressed by more specific or simpler attributes. In oemof.thermal, some of the technologies have a facade class
that can be found in the module oemof.thermal.facades. See the
```

Comment (jnnr) [reply docs/getting_started.rst]: There is no facade for the heatpump in oemof.thermal, e.g.

Comment (c-moeller) [reply docs/getting_started.rst]: Oh yes, you are right. Then this text was outdated anyway..

**Pull request merged**